src/globalContext.ts

```diff
@@ -52,32 +52,35 @@ export async function getGlobalContext(
 
     const {
         __STORYBOOK_CLIENT_API__: {
-            store
+            store,
+            storyStore
```

Comment:
Which one is supported on which version?

Reply:
`store` is supported on <=6.3. I'm not sure `storyStore` is support by which versions but it's supported on 6.4.

Should we switch the order of expressions or add a comment for this?

Reply:
Hmm, older versions had `_storyStore`, which I was using in CLI plugin. It would be better if you could annotate each parameter with a comment for future reference.

Reply:
Actually, there is also `_storyStore`; however, since it starts with an underscore, it looks like a private field. That's why I didn't use `_storyStore`. I can change to `_storyStore` and remove both fields.